.github/CODEOWNERS

```diff
@@ -1,6 +1,7 @@
-**/*-native/*	@mendix/native-content
+* @mendix/native-content @mendix/web-content
```

Comment:
Will this be overruled by the other rules?

Reply:
This will be overridden. It means if something is changed in -web widgets, just web will be required, but if something touches tools and atlas both teams needs to approve

Reply:
I will make the rules a bit more clear

Reply:
Done

Reply:
Rules are being overridden according to the order declared.